### ISSUE TITLE
Option to enable built-in or mod-provided data and resource packs

### ISSUE THREAD

**Issue by dhouck** [JTK222/Global-Packs#48]
The vanilla game provides feature toggle datapacks, as well as programmer art and high-contrast resource packs.  Mods can also provide optional resource packs, and I believe also optional data packs.  It would be nice to be able to auto-enable those with this mod.

**Comment by JTK222** [comment]
I can take a look at the optional vanilla packs, but I don't think I'll be able to do the same for mods.
There is no real standard to my knowledge that works across all modloaders, and I am not willing to make features that will only ever work for one modloader.

**Comment by dhouck** [comment]
Fair enough; my main goal when suggesting this was to force-enable bundles and then I had scope creep when making the request.

**Comment by JTK222** [comment]
Custom datapacks can also enable feature flags, you'd still have to provide the recipes though.
I think that feature is part of the .mcmeta?

**Comment by dhouck** [comment]
Oh, yeah, I read that when the feature toggles first came out but I forgot.  Thanks

**Comment by JTK222** [comment]
Going to close this, as it's unfortunately not possible. If you want to enable feature flags, you can always create a custom datapack for that.
